service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractMapperTest.java

```diff
@@ -68,9 +68,9 @@
 
     @BeforeAll
     public static void initialize() throws IOException {
-        inputJsonFileContent = getFileContent(INPUT_PATH + JSON_INPUT_FILE);
-        inputJsonFileWithNoPatientContent = getFileContent(INPUT_PATH + JSON_INPUT_FILE_WITH_NO_PATIENT);
-        expectedJsonToXmlContent = getFileContent(OUTPUT_PATH + EXPECTED_XML_TO_JSON_FILE);
+        inputJsonFileContent = ResourceTestFileUtils.getFileContent(INPUT_PATH + JSON_INPUT_FILE);
```

Comment:
could use static import here

Reply:
I disagree. Having name of a static method's class at the front improves readability. If we have simply a static method called "prepareData" would not give us enough information. However if we have a class called "PatientDataProvider" and we invoke this method from it it would give us more information. If we only have "getFileContent", sure it gives us information about file content. However with "ResourceTestFileUtils" we know that we are getting the test file content.